src/Client/Traits/ClientTrait.php

```diff
@@ -36,6 +47,7 @@
      */
     public function send(Request $request)
     {
+        $this->monitor()->start($this->getMetricName(), ['request_name' => $request->getServiceKey().'.'.$request->getRequestKey()]);
```

Comment:
Can you split the request_name in two tags?

Reply:
sure!